engine/apps/api/views/telegram_channels.py

```diff
@@ -19,11 +19,14 @@ class TelegramChannelViewSet(
     viewsets.GenericViewSet,
 ):
     authentication_classes = (PluginAuthentication,)
-    permission_classes = (IsAuthenticated, ActionPermission)
-
-    action_permissions = {
-        IsAdmin: (*MODIFY_ACTIONS, "set_default"),
-        AnyRole: READ_ACTIONS,
+    permission_classes = (IsAuthenticated, RBACPermission)
+
+    rbac_permissions = {
+        # TODO: what permissions should go here?
+        "list": [],
+        "retrieve": [],
+        "destroy": [],
+        "set_default": [],
     }
```

Comment:
I assume the proper permission here would be `RBACPermission.Permissions.CHATOPS_<READ|WRITE>`?

Reply:
chatops